hazelcast-code-generator/src/main/java/com/hazelcast/client/protocol/generator/Lang.java

```diff
@@ -40,6 +40,11 @@
      *PYTHON lang
      */
     PY,
+    
+    /**
+     *PYTHON lang
```

Comment:
@bilalyasar you have duplicate PYTHON lang.
Also, personally I don't like `NODE` as language. `JS` or `JavaScript` would be better.